services/auth.py

```
@@ -85,6 +125,53 @@ def _save_github_token(self, token: GitHubToken):
+    def _load_discord_credentials(self) -> Optional[DiscordCredentials]:
+        """Load Discord credentials from cache."""
+        if not self.discord_credentials_file.exists():
+            return None
+        
+        try:
+            with open(self.discord_credentials_file, 'r') as f:
+                data = json.load(f)
+                # Convert string back to datetime
+                data['created_at'] = datetime.fromisoformat(data['created_at'])
+                return DiscordCredentials(**data)
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)